xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -406,16 +406,19 @@ Rail transport, Roads and Road transport
 	</xsd:group>
 	<xsd:group name="LinePropertiesGroup">
 		<xsd:annotation>
-			<xsd:documentation>Property Elements of a LINE.</xsd:documentation>
+			<xsd:documentation>Property elements of a LINE.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:choice minOccurs="0">
+			<xsd:element ref="AuthorityRef" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Primary OPERATOR of LINE.</xsd:documentation>
+					<xsd:documentation>A line can only have no Authority when this information is provided in a ResponsibilitySet.</xsd:documentation>
 				</xsd:annotation>
-				<xsd:element ref="AuthorityRef"/>
-				<xsd:element ref="OperatorRef"/>
-			</xsd:choice>
+			</xsd:element>
+			<xsd:element ref="OperatorRef" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Operators should be set, even when the same as the Authority. Can be omitted when provided via ResponsibilitySet</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
 			<xsd:element name="additionalOperators" type="transportOrganisationRefs_RelStructure" minOccurs="0">
```

Comment:
I think additionalOperators where added when we had a choice of Authority or Operator. Should we discuss this too?

Reply:
my comment ok?